app/renderer/components/common/switchControl.js

```diff
@@ -131,6 +131,7 @@ class SwitchControl extends ImmutableComponent {
               styles.switchControl__text__label,
               styles.switchControl__text__label_right,
               this.props.small && styles.switchControl__text__label_small,
+              this.props.disabled && styles.switchControl__text_disabled,
```

Comment:
@luixxiul, thank you for the feedback.
I am not very familiar with styling in this app or styling in React, would you mind telling me how does order affect styling? I tried reading https://github.com/brave/browser-laptop/blob/master/docs/style.md but can't find how order affect styling.

Would be great if you can suggested some reading about styling in React, thanks 😄

Reply:
hey @user512. The order affects styles the same way as CSS cascading works. The last rule overrides the previous so in your changes, the disabled interface would override other styles, which is the correct and expected behavior. Nice work!

Reply:
maybe this could be the last conditional but I think it's ok as-is. Thanks!